server/src/test/java/io/druid/initialization/InitializationTest.java

```diff
@@ -414,17 +415,15 @@ public void testGetURLsForClasspath() throws Exception
                 + File.pathSeparator
                 + tmpDir2.getAbsolutePath() + File.separator + "*";
 
-    List<URL> expected = ImmutableList.<URL>builder()
-                                      .add(tmpDir1a.toURI().toURL())
-                                      .add(tmpDir1b.toURI().toURL())
-                                      .add(tmpDir3.toURI().toURL())
-                                      .add(tmpDir2c.toURI().toURL())
-                                      .add(tmpDir2d.toURI().toURL())
-                                      .add(tmpDir2e.toURI().toURL())
-                                      .build();
+    // getURLsForClasspath uses listFiles which does NOT guarantee any ordering for the name strings.
+    List<URL> urLsForClasspath = Initialization.getURLsForClasspath(cp);
```

Comment:
will it work if we converted the "expected" list into a set and just did one assertEquals as before?